gymfc.ini

```diff
@@ -3,12 +3,12 @@
 # In most cases this should not be modified
 [DEFAULT]
 
-SetupFile = /usr/share/gazebo-9/setup.sh
```

Comment:
This is unique to your workspace so exclude this. We need a better way to find this file because this continues to be a problem for VC. I think it's best to pin a gazebo version to GymFC and set this version according to the install instructions, i.e, $HOME/local/share/gazebo-10/setup.sh

Reply:
This is actually the path that Gazebo is located in the Docker container. HOME happens to be /home/gymfc and the make install og Gazebo puts in _gazebo_

If we don't set this the docker image won't work, please advise

Reply:
I see, that makes sense. I just reviewed the code and I believe environment variables in SetupFile will actually work. Could you test if `SetupFile = $HOME/local/share/gazebo/setup.sh` works?

Still unsure why mine is installed in gazebo-10 but maybe the installer changed.

Reply:
I actually doubled checked on this and  for some reason the install makes two directories, one called gazebo and another gazebo-[version number]. Accessing the gazebo folder is a better option as you had it.